sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/HistogramAccumulation.java

```diff
@@ -37,6 +44,22 @@ static HistogramAccumulation create(double sum, long[] counts, List<ExemplarData
    */
   abstract double getSum();
 
+  /**
+   * The min of all measurements recorded.
+   *
+   * @return the min of recorded measurements, or {@code null} if not available.
+   */
+  @Nullable
```

Comment:
Since this is internal, it's probably worth having it match our expectations. They appear to be `boolean hasMinMax()` and unboxed for the others, since we always have both min/max or neither. Probably same or less code than the `applyToNullable` pattern if doing that too